runtime/mangata-kusama/src/lib.rs

```diff
@@ -1116,7 +1171,7 @@ construct_runtime!(
 		XTokens: orml_xtokens::{Pallet, Storage, Call, Event<T>} = 35,
 		UnknownTokens: orml_unknown_tokens::{Pallet, Storage, Event} = 36,
 		OrmlXcm: orml_xcm::{Pallet, Call, Event<T>} = 37,
-		AssetRegistry: xcm_asset_registry::{Pallet, Call, Storage, Event, Config} = 38,
+		AssetRegistry: orml_asset_registry::{Pallet, Call, Storage, Event<T>, Config<T>} = 38,
```

Comment:
maybe its better to use some other id ? (not sure actually)

Reply:
me neither, i think we should be fine
the old storage is cleared in migration